driver/razermousemat_driver.c

```diff
@@ -637,6 +652,19 @@ static int razer_mousemat_probe(struct hid_device *hdev, const struct hid_device
     struct usb_interface *intf = to_usb_interface(hdev->dev.parent);
     struct usb_device *usb_dev = interface_to_usbdev(intf);
     struct razer_mousemat_device *dev = NULL;
+    unsigned char expected_protocol = USB_INTERFACE_PROTOCOL_MOUSE;
+
+    switch(usb_dev->descriptor.idProduct) {
+    case USB_DEVICE_ID_RAZER_FIREFLY_HYPERFLUX:
+    case USB_DEVICE_ID_RAZER_FIREFLY:
+    case USB_DEVICE_ID_RAZER_GOLIATHUS_CHROMA:
+    case USB_DEVICE_ID_RAZER_GOLIATHUS_CHROMA_EXTENDED:
+        expected_protocol = USB_INTERFACE_PROTOCOL_MOUSE;
+        break;
+    case USB_DEVICE_ID_RAZER_FIREFLY_V2:
+        expected_protocol = RAZER_FIREFLY_V2_INTERFACE_PROTOCOL;
```

Comment:
just put `0` here instead of hiding it behind `RAZER_FIREFLY_V2_INTERFACE_PROTOCOL`

Reply:
Fixed